grpc/cargo_casper/tests/integration_tests.rs

```diff
@@ -71,6 +71,9 @@ fn run_tool_and_resulting_tests(turbo: bool) {
     test_cmd.arg("test").current_dir(test_dir.join("tests"));
     let test_output = output_from_command(test_cmd);
     assert_eq!(SUCCESS_EXIT_CODE, test_output.status.code().unwrap());
+
+    // Cleans up temporary directory, but leaves it otherwise if the test failed.
+    fs::remove_dir_all(&temp_dir).unwrap();
```

Comment:
I missed this over the comment, this is backwards - there is no need to remove a `TempDir` instance, it will always be erased.

Reply:
Yes, that's the point though - usually I'm working on contract APIs, or some EE internals and whenever it will induce a test failure here I'll still be able to inspect the build manually rebuilding manually inside /tmp. Happy path will always clean up though

Reply:
Sounds good to me then.